tfx/components/example_gen/driver_test.py

```diff
@@ -100,18 +103,69 @@ def testResolveExecProperties(self):
         """
         splits {
           name: "s1"
-          pattern: "span02/split1/*"
+          pattern: "span02/version01/split1/*"
         }
         splits {
           name: "s2"
-          pattern: "span02/split2/*"
+          pattern: "span02/version01/split2/*"
         }""", updated_input_config)
 
+    # Reset exec properties to original pattern.
+    self._exec_properties = {
+        utils.INPUT_BASE_KEY:
+            self._input_base_path,
+        utils.INPUT_CONFIG_KEY:
+            json_format.MessageToJson(
+                example_gen_pb2.Input(splits=[
+                    example_gen_pb2.Input.Split(
+                        name='s1',
+                        pattern='span{SPAN}/version{VERSION}/split1/*'),
+                    example_gen_pb2.Input.Split(
+                        name='s2',
+                        pattern='span{SPAN}/version{VERSION}/split2/*')
+                ]),
+                preserving_proto_field_name=True),
+    }
+    
+    # Write new versions of files.
```

Comment:
I mean just have a v2 above, just run executor once is fine

(like above we have span1 and span2, span2 is picked up)

a single test can cover `pick up latest version of latest span`